### ISSUE TITLE
Create a homepage for the API

### ISSUE THREAD

**Issue by vitokhangnguyen** [Haider8/tmessage-api#4]
Currently, the API is hosted [here](https://peaceful-waters-15584.herokuapp.com/) without a proper homepage.
It would be nice that the API can have a homepage introduce the project and, probably, summarize all the possible routes of it.

**Comment by Haider8** [comment]
@vitokhangnguyen We can use [Aglio](https://github.com/danielgtaylor/aglio) and [Dredd](https://dredd.org/en/latest/) for this

**Comment by vitokhangnguyen** [comment]
Definitely! I am currently not having much time for it so I think it is a good idea to put a `Hacktoberfest` tag on it so someone can contribute for their PRs.
